docker-compose.yml

```diff
@@ -17,7 +17,6 @@ services:
       - LIGHTDASH_INSTALL_ID=${LIGHTDASH_INSTALL_ID}
       - LIGHTDASH_INSTALL_TYPE=${LIGHTDASH_INSTALL_TYPE:-docker_image}
     volumes:
-      - "${DBT_PROFILES_DIR}:/usr/app/profiles"
```

Comment:
same with this - is this meant to be in this PR?

Reply:
We no longer use the profiles.yml from their local machine so this volume isn't necessary anymore.

Reply:
So new default is to use the UI and not to use a local dbt project?

Reply:
> So new default is to use the UI and not to use a local dbt project?
@owlas The default and only option to setup the warehouse connection is via the UI, using a local dbt project or not.

Reply:
Ah yes! This is just the profiles dir - TY